cve_bin_tool/data_sources/__init__.py

```diff
@@ -1,9 +1,15 @@
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+import sys
```

Comment:
```suggestion
from __future__ import annotations

import sys
```

I think this should solve the problem on 3.7, will merge and try.